oak/server/oak_node.h

```diff
@@ -59,16 +64,7 @@ class OakNode final : public Node::Service {
   // Incoming gRPC data for the current invocation.
   const ::grpc::GenericServerContext* server_context_;
 
-  // Guards data used by OakRead and OakWrite host methods.
-  absl::Mutex module_data_mutex_;
```

Comment:
Because all the interactions with channels are effectively serialized by the fact that we are executing single-thread code in a single-thread gRPC loop.